### DISCUSSION TITLE
Building \ Downloading gitextentions for linux  later v2.51

### DISCUSSION THREAD

**Discussion by arthurvaverko-kaltura** [gitextensions/gitextensions#8538] [Q&A]
I have used [this wiki page ](https://github.com/gitextensions/gitextensions/wiki/How-To:-run-Git-Extensions-on-Linux) to download and lunch git extentions from Ubuntu and it works perfectly.
but its a very old version.

any pre-compiled versions ? or any up to date instruction on compiling with mono ?

**Comment by RussKie** [comment]
[2.51.05](https://github.com/gitextensions/gitextensions/releases/download/v2.51.05/GitExtensions-2.51.05-Mono.zip) is the last version we officially built for non Windows distros that use Mono.

I don't think any of the newer versions will work on Linux, but I have never tried either.

**Reply by arthurvaverko-kaltura** [reply]
well, i am trying now, I love the tool, and recently switched to linux, don't want to drop it for now :)`
the only issue i have so far with compiling the source is resources that are not found 
like this 

![image](https://user-images.githubusercontent.com/3643617/95818277-237c4180-0d2c-11eb-9f8c-a064ae294dd1.png)

**Reply by RussKie** [reply]
Which solution file are you using? GitExtensionsMono.sln?

It has been few years since I last run Ubuntu and Rider, but it compiled just fine in VS 2019:
![image](https://user-images.githubusercontent.com/4403806/96362455-21c1dc00-1179-11eb-8f57-3895217df4de.png)

**Reply by RussKie** [reply]
Is there any reason you can't use the compiled version, and want to build it yourself?

**Reply by arthurvaverko-kaltura** [reply]
Is there a compiled binary with mono ? 
I'm running ubuntu can just open the exe

**Comment by RussKie** [comment]
I provided the link above.

**Reply by arthurvaverko-kaltura** [reply]
Haa you mean 2.51 version.. its quite old .. I was wondering if I could compile the latest release with mono .. this is what I'm trying to do ..

**Reply by gerhardol** [reply]
Mono is only supported with the 2.x version. (Mono WinForms has hardly been updated in 10 years so that was just impossible to support.)
Some discussions to run GE in WINE (problem seem to be how to wrap Git).

**Reply by arthurvaverko-kaltura** [reply]
Okay will check that out .. thanks ..

**Comment by RussKie** [comment]
To be honest I would be surprised if the latest version compiles, let alone
works...
But it is certainly an interesting experiment. You can drop all plugins as
the first step, and see if the app compiles and runs.
That's how I started the port to .NET 5.0.
